src/styles/blocks/card.scss

```diff
@@ -0,0 +1,69 @@
+.card {
+  @include cardSize($cardWidth);
```

Comment:
I meant https://en.bem.info/forum/4/ mixes not mixins)

Reply:
I can't create mixes 
![image](https://user-images.githubusercontent.com/36269821/197390132-36cf87aa-db94-4fff-b3ee-bcbeaa0d1cee.png)

Reply:
I do not understand what to do.